esp/esp/program/modules/handlers/studentlunchselection.py

```diff
@@ -120,6 +120,7 @@ def isCompleted(self):
     @main_call
     @needs_student
     @meets_cap
+    @meets_deadline('/Classes')
```

Comment:
I'm actually not sure whether this change is a good thing or not. It seems like it fixes some use cases and breaks others.